src/helpers/patientUtils.js

```diff
@@ -147,10 +148,33 @@ function maskPatientData(bundle, mask) {
   });
 }
 
+/**
+ * Mask all references to the MRN used as an id
+ * Currently, the MRN appears as an id in 'subject' and 'individual' objects in other resources
+ * and in the 'id' and 'fullUrl' fields of the Patient resource.
+ * Replaces the MRN with a hash of the MRN
+ * @param {Object} bundle a FHIR bundle with a Patient resource and other resources
+ */
+function maskMRN(bundle) {
+  const patient = fhirpath.evaluate(bundle, 'Bundle.entry.where(resource.resourceType=\'Patient\')')[0];
+  if (patient === undefined) throw Error('No Patient resource in bundle. Could not mask MRN.');
+  const mrn = patient.resource.id;
+  const masked = shajs('sha256').update(mrn).digest('hex');
+  patient.fullUrl = `urn:uuid:${masked}`;
+  patient.resource.id = masked;
+  const subjects = fhirpath.evaluate(bundle, `Bundle.entry.resource.subject.where(reference='urn:uuid:${mrn}')`);
+  const individuals = fhirpath.evaluate(bundle, `Bundle.entry.resource.individual.where(reference='urn:uuid:${mrn}')`);
+  const mrnOccurances = subjects.concat(individuals);
```

Comment:
```suggestion
  const mrnOccurrences = subjects.concat(individuals);
``` 

Heheh sorry. Would also have to update the uses of it below